packages/cli/test/e2e/runDevCmd.test.ts

```diff
@@ -9,11 +9,12 @@ describeCliTest("Run dev command", function ({spawnCli}) {
   itDone("Run dev command with no --dataDir until beacon api is listening", async function (done) {
     const beaconPort = 39011;
 
-    const devProc = spawnCli({pipeStdToParent: false, printOnlyOnError: true, logPrefix: "dev"}, [
+    const devProc = spawnCli({pipeStdToParent: true, printOnlyOnError: true, logPrefix: "dev"}, [
```

Comment:
Warning we do not want to pollute test output. If you need that enable with a temporary flag on ENV variable. Or upload output in a test artifact

Reply:
whoops, stray diff from debugging these tests

Reply:
should be removed in latest commit of this branch